src/styles/shadow.css

```
@@ -1,6 +1,29 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
+
+@config '../../tailwind.config.js';
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@config`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)